af_odoo_tour/wizard/intoduction_wizard.py

```diff
@@ -107,17 +107,39 @@ def _get_records(self):
     def search_jobseeker(self):
         # TODO: This should be made into two separate functions so it's 100%
         # clear what the user is trying to do.
+        now_year = str(datetime.now().year)
+        before_year = str(datetime.now().year - 100)
```

Comment:
before_year is a pretty ambiguous variable name, consider changing to last_century or something more descriptive.